dialogueMaker/dialogueMaker.cpp

```diff
@@ -210,3 +214,15 @@ std::string Game::parseText(std::string text) {
   }
   return parsedText;
 }
+
+void Game::addPlayer(player p){
+  Player = p;
+}
+
+void Game::printstats(){
+  Player.printstats();
+
+}
+
+player Game::Player;
```

Comment:
Necessary?

Reply:
![image](https://github.com/BallballTH/Tom-Khai/assets/49591895/cd86827f-4e27-4302-89c6-856906231985)
this throw up if delete player Game::Player;